Userland/Libraries/LibPDF/Fonts/Type1FontProgram.cpp

```diff
@@ -250,16 +253,17 @@ PDFErrorOr<Type1FontProgram::Glyph> Type1FontProgram::parse_glyph(ReadonlyBytes
         int v = data[i];
         if (v == 255) {
             TRY(require(4));
-            int a = data[++i];
-            int b = data[++i];
-            int c = data[++i];
-            int d = data[++i];
+            // Both Type 1 and Type 2 spec:
+            // "If the charstring byte contains the value 255, the next four bytes indicate a two’s complement signed number.
+            //  The first of these four bytes contains the highest order bits [...]"
+            i32 a = static_cast<i32>((data[i + 1] << 24) | (data[i + 2] << 16) | (data[i + 3] << 8) | data[i + 4]);
```

Comment:
Could maybe use `i32 a = BigEndian<i32>(*reinterpret_cast<i32*>(data));` here?

Reply:
Funny way to write `bit_cast`

Reply:
I had that at first, but I liked it less /shrug

Reply:
Personally I like the readability of "take the big endian i32 at this location" vs. having to check byte indices and shift sizes. But feel free to ignore, someone else might do a big sweep over the codebase one day to replace these occurrences anyway :^)

Reply:
:yakshed: Maybe we want a `template<typename T> T& Bytes::value_at(offset)` method to make this nicer to do?

Reply:
This PR is about fixing a bug in existing code and uses code similar in style to the code the bug is in. Maybe this discussion could happen somewhere that isn't this PR? 😅

Reply:
> Maybe this discussion could happen somewhere that isn't this PR?

Fair.